src/extensions/default/CodeFolding/Prefs.js

```diff
@@ -23,7 +22,7 @@ define(function (require, exports, module) {
                            {name: strings.SAVE_FOLD_STATES, description: strings.SAVE_FOLD_STATES_HELP});
     prefs.definePreference("alwaysUseIndentFold", "boolean", true,
                            {name: strings.ALWAYS_USE_INDENT_FOLD, description: strings.ALWAYS_USE_INDENT_FOLD_HELP});
-    prefs.definePreference("enableRegionFolding", "boolean", true,
+    prefs.definePreference("enableRegionFolding", "boolean", false,
```

Comment:
Ok got it. The custom region folds feature is being removed. In that case, could't we remove the preference entry altogether, along with the menu entry and the corresponding strings?